learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md

```diff
@@ -62,6 +62,8 @@ The VS Code Ballerina extension gives you the same debugging experience as the c
 
 Then, you can start a program, test, or remote debug session as shown below.
 
+>**Info:** The working directory will always be the package root if the user starts the debug session through VS Code. Users can use remote debugging for alternative working directories.
```

Comment:
```suggestion
>**Info:** For the debug sessions which get launched from the VS Code, the working directory will be the Ballerina package root. Users can use remote debugging for alternative working directories.
```

Reply:
@praneesha Appreciate your input on this

Reply:
Replied below.